--- origsrc/gjs-1.30.0/Makefile-test.am	2011-07-25 12:54:00.000000000 -0500
+++ src/gjs-1.30.0/Makefile-test.am	2011-12-11 02:24:41.748957200 -0600
@@ -2,7 +2,7 @@ RUN_WITH_DBUS = ${top_srcdir}/test/run-w
 GTESTER = ${TESTS_ENVIRONMENT} ${RUN_WITH_DBUS} gtester
 
 ########################################################################
-TEST_PROGS += gjs-tests
+TEST_PROGS += gjs-tests$(EXEEXT)
 
 gjs_tests_CFLAGS =				\
 	-include $(top_srcdir)/test/test.h	\
@@ -37,7 +37,7 @@ gjstest.h.stamp : scripts/make-tests $(g
 gjstest.h gjstest.c : gjstest.h.stamp
 	@true
 
-TEST_PROGS += gjs-unit
+TEST_PROGS += gjs-unit$(EXEEXT)
 
 gjs_unit_CPPFLAGS =		\
 	$(AM_CPPFLAGS)          \
