--- origsrc/gjs-1.43.3/Makefile-insttest.am	2014-12-21 11:56:52.000000000 -0600
+++ src/gjs-1.43.3/Makefile-insttest.am	2015-11-08 14:27:03.115793800 -0600
@@ -53,7 +53,7 @@ endif
 # This rpath /nowhere thing is the libtool upstream recommended way to
 # force generation of shared libraries, which we need in order for the
 # tests to work uninstalled.
-common_test_ldflags = -avoid-version -rpath /nowhere
+common_test_ldflags = -avoid-version -rpath /nowhere -no-undefined
 common_test_libadd = $(GJS_LIBS)
 
 nodist_libregress_la_SOURCES = $(GI_DATADIR)/tests/regress.c $(GI_DATADIR)/tests/regress.h
